### PR TITLE
Scopes improvement

### PR DESCRIPTION
**Pull request by albertmourato**
Providing detailed information about `WEDEPLOY_AUTH_SECURE_FIELDS` on auth environment variables, explaining how to set a user scope, and how to filter data access by scope

### CODE REVIEW

File: node/electric/src/pages/docs/auth/environment-variables/index.md [hunk 1]
```diff
@@ -43,4 +43,30 @@ Here's a list of all the environment variables you can use with this service.
...
+
+Please remember that supportedScopes works differently than the rest. To learn more, see our [Data Validation docs](/docs/data/configuring-data/#5).
+
+```application/json
+{
+	"id": "auth",
+	"image": "wedeploy/auth:2.0.0",
+	"env": {
+		"WEDEPLOY_AUTH_SECURE_FIELDS": "providers,password,resetKey,supportedScopes",
+		"WEDEPLOY_AUTH_PASSWORD": "true",
```
Comment (eduardolundgren) [inline node/electric/src/pages/docs/auth/environment-variables/index.md]: Remove trailing comma here

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 2]
```diff
@@ -355,6 +355,59 @@ WeDeploy
 
 </article>
 
+<article id="9">
+	
+## Set User Scopes
+
+User scopes allows you to choose what kind of role or access you provide to each user. This could include things like grouping team users or providing admin access to your app. In order to setup user scopes, you must set a `supportScope` when creating or updating a user.
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: User scopes allows > User scopes allow
you must set a > you must declare a

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 3]
```diff
@@ -355,6 +355,59 @@ WeDeploy
...
+
+**Creating user with user roles**
+
+```javascript
+WeDeploy
+	.auth('https://<serviceID>-<projectID>.wedeploy.io')
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: They must also add credentials, like a master token, to this request, right?

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 4]
```diff
@@ -355,6 +355,59 @@ WeDeploy
...
+	.catch(function(err) {
+		// Not created.
+	});
+```
+
+**Updating user with user roles**
+
+```javascript
+WeDeploy
+  .auth('https://<serviceID>-<projectID>.wedeploy.io')
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: Credentials here too.

File: node/electric/src/pages/docs/auth/manage-users/index.md (hunk 3 above)
Comment (albertmourato) [reply node/electric/src/pages/docs/auth/manage-users/index.md]: Yes, good catch!

**Pull request merged**